describe_groups_response.go

```diff
@@ -0,0 +1,162 @@
+package sarama
+
+type DescribeGroupsResponse struct {
+	Groups []*GroupDescription
```

Comment:
I also like that this matches the request: you ask for a slice of groups, you get a slice of group descriptions back.